bin/old_scones.nf

```diff
@@ -10,7 +10,7 @@ fam = file("${bed.getParent()}/${bed.getBaseName()}.fam")
 // SConES parameters
 params.network = 'gs'
 params.score = 'chi2'
-params.criterion = 'consistency'
+params.criterion = 'stability'
```

Comment:
This is my fault for making it a bit confusing. I have two SConES pipelines: run_scones (should be renamed to scones.nf) and old_scones.nf. The latter is the one whose implementation I tested really well, and uses the function search_cones. The former was not that well tested, and uses the more modern function scones.cv. I think all the changes on old_scores.nf should be reverted, and we should focus on run_scones instead. We should verify if we obtain the same results and, if so, stick to the modern pipeline.

Reply:
I emailed Vivien to ask if he checked the equivalence of the old and the new version